server/db/models/Developer.js

```diff
@@ -0,0 +1,161 @@
+const mongoose = require('mongoose'),
+  bcrypt = require('bcryptjs'),
+  jwt = require('jsonwebtoken');
+
+const developerSchema = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      trim: true,
+      unique: true,
+    },
+    school: {
+      type: String,
+      trim: true,
+      unique: true,
```

Comment:
should the school be unique? if it's unique, then we wouldn't be able to have the same school on multiple developers, right?

Reply:
I think role was front-end/ back-end/ full-stack, where as teamLead would be whether or not a developer wanted to be a PM or not.  But yes, unique should not be a a truthy attribute for either of them!